Dockerfile

```diff
@@ -0,0 +1,32 @@
+# syntax=docker/dockerfile:experimental
+
+FROM opensuse/tumbleweed:latest
+RUN zypper ref && zypper up -y
+#RUN zypper in -y azure-cli
+
+# way suggested on https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=zypper
+RUN zypper install -y tar gzip unzip curl python310-pip openssh
+RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc
+RUN zypper addrepo --name 'Azure CLI' --check https://packages.microsoft.com/yumrepos/azure-cli azure-cli
+RUN zypper install --from azure-cli -y azure-cli
+RUN zypper clean --all
+
+
+WORKDIR /root
+RUN curl https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-387.0.0-linux-x86_64.tar.gz -o google-cloud-cli.tar.gz
+RUN tar xvf google-cloud-cli.tar.gz
+RUN google-cloud-sdk/install.sh --quiet --usage-reporting false --command-completion true
+RUN echo 'source /root/google-cloud-sdk/completion.bash.inc' >> ~/.bashrc
+RUN echo 'source ~/google-cloud-sdk/path.bash.inc' >> ~/.bashrc
+RUN rm google-cloud-cli.tar.gz
+
+RUN curl https://releases.hashicorp.com/terraform/1.1.7/terraform_1.1.7_linux_amd64.zip -o terraform.zip
```

Comment:
Ditto above.

Reply:
Yeah, it looks super messy, but for a small image size you can do something like:

```dockerfile
  RUN tar xvf google-cloud-cli.tar.gz && \
  google-cloud-sdk/install.sh --quiet --usage-reporting false --command-completion true && \
  echo 'source /root/google-cloud-sdk/completion.bash.inc' >> ~/.bashrc && \
  RUN echo 'source ~/google-cloud-sdk/path.bash.inc' >> ~/.bashrc && \
  RUN rm google-cloud-cli.tar.gz
```

Reply:
Test result
```
podman images
REPOSITORY                                                                                     TAG                                   IMAGE ID      CREATED             SIZE
localhost/qeqe-dev-small                                                                       latest                                6c3ea9e4b938  About a minute ago  2.09 GB
localhost/qeqe-dev                                                                             latest                                562a8ece21c5  2 days ago          2.24 GB
```